release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -559,6 +559,13 @@ You can now create a password for the {op-system} `core` user. If you cannot use
 
 For more information, see xref:../post_installation_configuration/machine-configuration-tasks.adoc#core-user-password_post-install-machine-configuration-tasks[Changing the core user password for node access].
 
+[id="ocp-4-13-management-console"]
+=== Management console
```

Comment:
@dfitzmau these should be included with the other management console bugs in the bug fix section.

Reply:
Joe addressed these bugs in https://github.com/openshift/openshift-docs/pull/59751/files.